spring-grpc-core/src/main/java/org/springframework/grpc/server/service/DefaultGrpcServiceConfigurer.java

```diff
@@ -44,8 +45,8 @@ public DefaultGrpcServiceConfigurer(ApplicationContext applicationContext) {
 		this.applicationContext = applicationContext;
 	}
 
-	@PostConstruct
```

Comment:
Did not want to mess w/ bringing in jakarta-annotations-api in core (it was dep mgmt in autoconfigure module via SB).